src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+
+    private Random random = new Random();
+    private String[] myFigure = {"square","rectangle","right triangle","circle",
+            "isosceles trapezoid"};
+
+    public void getRandomFigure(int figureNumber) {
```

Comment:
тут з методу потрібно повертати фігуру щоб можна було викликати метод і присвоїти результат його роботи у якусь змінну і працювати далі з цією змінною. Також нам не потрібно нічого передавати в цей метод, його задача просто видати якусь рандомну фігуру, йому потрібно самому внутрішньо вирішувати як це зробити
```suggestion
    public Figure getRandomFigure() {
```